[DOC] Pyroscope landing page for doc (#3176)
grafana/pyroscope: docs/sources/_index.md

  - profiles datastore
  - observability
  - continuous profiling
hero:
  title: Grafana Pyroscope
  level: 1
  image: /static/img/pyroscope-logo.svg
  width: 110
  height: 110
  description: >-

>     Grafana Pyroscope is an open source software project for aggregating continuous profiling data. Continuous profiling is an observability signal that allows you to understand your workload's resources usage down to the line number.

Reviewer: Not too sure if that makes it better or not.

```suggestion
    Grafana Pyroscope is an open source software project for aggregating continuous profiling data. Continuous profiling is an observability signal that allows you to understand your workload's resources usage down to the source code line number.
```

    - title: Pyroscope architecture
      href: ./reference-pyroscope-architecture/
      description: Take a deep-dive into the microservices-based architecture to learn about deployment modes, components (microservices), and more. The system has multiple horizontally scalable microservices that can run separately and in parallel.
---

{{< docs/hero-simple key="hero" >}}

---

## Overview

Grafana Pyroscope is a multi-tenant, continuous profiling aggregation system, aligning its architectural design with Grafana Mimir, Grafana Loki, and Grafana Tempo.
It facilitates the ingestion, storage, and querying of profiles and integrates with Grafana.
This integration enables a cohesive correlation of profiling data with existing metrics, logs, and traces.

Continuous profiling provides ongoing, in-depth insights into application performance.
With Pyroscope, you go from a bunch of flame graph files on your desktop to a database you can query and analyze production data in a structured way.

<video style="border-radius: 1%; width: 75%; display: block; margin-left: auto; margin-right: auto;" autoplay loop>

>   <source src="ui.webm" type="video/webm">

Reviewer: It probably makes sense to update that video. It shows a quite outdated version of explore. We probably want to show either the new explore (or the app shipped with Pyroscope).

Author: I replace the video with this screenshot: 
![image](https://github.com/grafana/pyroscope/assets/104772500/6a31df78-213c-43a2-9b72-05b6d3f9b3f1)

> <video style="border-radius: 1%; width: 75%; display: block; margin-left: auto; margin-right: auto;" autoplay loop>
>   <source src="ui.webm" type="video/webm">
> </video>

Author: ```suggestion
<!--video style="border-radius: 1%; width: 75%; display: block; margin-left: auto; margin-right: auto;" autoplay loop>
  <source src="ui.webm" type="video/webm">
</video-->
![Pyroscope UI showing the comparison view](/media/docs/pyroscope/screenshot-pyroscope-comp-view.png)
```

    - title: Configure the server
      href: ./configure-server/
      description: Configure your Pyroscope server to meet your needs by setting disk storage, tenant IDs, memberlist, proxies, shuffle sharding, and more. You can also use the server HTTP API.
    - title: View and analyze profile data
      href: ./view-and-analyze-profile-data/
      description: Profiling data can be presented in a variety of formats presents, including flame graphs, tables, as well as charts and graphs. Flame graphs visualize call relationships and identify hot spots. Tables let you view detailed statistics for specific functions or time periods. Charts and graphs help you analyze trends and compare performance across different metrics.
    - title: Pyroscope architecture
      href: ./reference-pyroscope-architecture/
      description: Take a deep-dive into the microservices-based architecture to learn about deployment modes, components (microservices), and more. The system has multiple horizontally scalable microservices that can run separately and in parallel.
---

{{< docs/hero-simple key="hero" >}}

---

## Overview

> Grafana Pyroscope is a multi-tenant, continuous profiling aggregation system, aligning its architectural design with Grafana Mimir, Grafana Loki, and Grafana Tempo.
> It facilitates the ingestion, storage, and querying of profiles and integrates with Grafana.
> This integration enables a cohesive correlation of profiling data with existing metrics, logs, and traces.
> 
> Continuous profiling provides ongoing, in-depth insights into application performance.
> With Pyroscope, you go from a bunch of flame graph files on your desktop to a database you can query and analyze production data in a structured way.

Author: ```suggestion
Grafana Pyroscope is a multi-tenant, continuous profiling aggregation system, aligning its architectural design with Grafana Mimir, Grafana Loki, and Grafana Tempo.
This integration enables a cohesive correlation of profiling data with existing metrics, logs, and traces.

Explore continuous profiling data to gain insights into application performance.
You can query and analyze production data in a structure way. 
Use the Pyroscope UI or Grafana to visualize the data. 
```
